public/app/features/dashboard/components/Inspector/PanelInspector.tsx

```diff
@@ -99,11 +108,12 @@ export class PanelInspectorUnconnected extends PureComponent<Props, State> {
    */
   init() {
     const { plugin, panel } = this.props;
+    const { withTransforms, withFieldConfig } = this.state;
 
     if (plugin && !plugin.meta.skipDataQuery) {
       this.querySubscription = panel
         .getQueryRunner()
-        .getData()
+        .getData({ withTransforms, withFieldConfig })
```

Comment:
This function is called many times, need to check if query subscription exists an unsubscribe before subscribing again